### ISSUE TITLE
Question about chrome "managed" storage

### ISSUE THREAD

**Issue by nicolasguilhem** [loilo/auto-group-tabs#37]
Hello,

Does this extension works with "managed" storage ?

I want to deploy extension configuration using `storage.managed` area (see [documentation on chrome.storage](https://developer.chrome.com/docs/extensions/reference/storage/#storage-areas) and [documentation on Chrome extensions manifest](https://developer.chrome.com/docs/extensions/mv3/manifest/storage/)).

It says : 

> It's up to the extension to enforce the policies configured by the administrator.

If this can work with this extension, how to send pre-configuration of the extenstion ?

Thanks

**Comment by loilo** [comment]
I have never worked with managed storage, so I haven't really considered it for this extension. I guess I could generally make it work though.

Just so I correctly understand what's the use case here: What you'd want to do is provide pre-configured, non-editable grouping rules to administrated Chrome instances?

**Comment by nicolasguilhem** [comment]
> What you'd want to do is provide pre-configured, non-editable grouping rules to administrated Chrome instances?

Yes, I want to use GPO to install extension automatically and provide pre-configured groups configuration.

The users, when lauching chrome, doesn't have to care about installing and configuring the extension.

**Comment by loilo** [comment]
Okay.

As far as I understand `managed` storage, this would be a better fit for a whole new extension, since managed storage is not editable by a user which would basically make all of the extension's UI obsolete.

Is that how you'd use it? Or rather like a pre-delivered configuration that user's could still adjust?

**Comment by nicolasguilhem** [comment]
Yes that how I want to use it, you're right. I don't want users to be able to change the configuration.

**Comment by loilo** [comment]
Okay, this definitely calls for a variant of the extension (something like "Auto Group Tabs Managed") that only uses the grouping logic with no settings UI whatsoever.

Let's see if I can make this work.
